### PR TITLE
Small changes and clarification.

### PR DESCRIPTION
**Pull request by pr0hin**
@RalfJung  When I was learning through this tutorial, I had some major confusion as to what function was being referred to in the lifetimes section [here](https://github.com/RalfJung/rust-101/compare/RalfJung:3067763...r0e:b89eed2#diff-b2ab04260fc4b9d88ffad90cf39d9c0cR140).

I also noticed some other tiny errors as I was reading and thought that I could fix them up for you.

Overall, the best tutorial I've read so far (especially the lifetimes section).

### CODE REVIEW

File: src/part02.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ pub enum SomethingOrNothing<T>  {
 }
 // Instead of writing out all the variants, we can also just import them all at once.
 pub use self::SomethingOrNothing::*;
-//@ What this does is to define an entire family of types: We can now write
+//@ What this does is define an entire family of types: We can now write
```
Comment (RalfJung) [inline src/part02.rs:20]: Are you a native speaker? I'm not, and I just want to be sure here ;-) . I thought one could do "is defining" and "is to define" here, but not "is define".

**Comment by RalfJung** [comment]
Thanks for the fixes! I'm glad you found the tutorial helpful :)

File: src/part02.rs (hunk 1 above)
Comment (pr0hin) [inline src/part02.rs:20]: Yup I'm a native speaker.
In this case I'm certain that "is define" works best here.

If you wanted to use "is defining" you'd want to write something along the lines of "What we are doing is defining an entire family of types"

**Comment by pr0hin** [comment]
There are also some instances in files where you say "You borrowed your friend a Vector" like [here](https://github.com/RalfJung/rust-101/blob/306776311fd5ec57a462ea227c40ba0acb2fec73/src/part06.rs#L129-L130).

It might be better to say "You loaned your friend" or "You lent your friend". Though I didn't include them in this commit because I wasn't sure whether you intentionally wrote "borrowed" for readers.

**Comment by RalfJung** [comment]
I said "borrowed" because that's the term Rust is using. Is it outright wrong to use this word here?

**Comment by pr0hin** [comment]
Ah I thought so. In this case, it is okay to leave it as it is.

**Pull request merged**